local-volume/helm/provisioner/templates/provisioner.yaml

```diff
@@ -1,39 +1,30 @@
 {{- define "configStorageClass" }}
-{{- $classPath := . }}
-{{- range $tmp, $val := $classPath }}
-   {{- range $storageClass, $classConfig := $val }}
-{{ $storageClass }}:
+{{- range $classConfig := . -}}
```

Comment:
Is it possible to put all this inline with the rest of the spec template instead of at the beginning?

Reply:
Done, I removed variables aliases which I found not necessary too. It's hard to read diff of provisioner.yaml now, you can simply check [final version of provisioner](https://github.com/cofyc/external-storage/blob/refactor_heml_chart/local-volume/helm/provisioner/templates/provisioner.yaml).

I also added a helm chart test script here: https://github.com/cofyc/external-storage/tree/refactor_heml_chart/local-volume/helm/test